lib/ansible/modules/clustering/kubevirt/kubevirt_vm.py

```diff
@@ -0,0 +1,312 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2018, KubeVirt Team <@kubevirt>
+# Apache License, Version 2.0
+# (see LICENSE or http://www.apache.org/licenses/LICENSE-2.0)
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: kubevirt_vm
+
+short_description: Manage KubeVirt virtual machine
+
+description:
+    - Use Openshift Python SDK to manage the state of KubeVirt virtual machines.
+
+version_added: "2.8"
+
+author: KubeVirt Team (@kubevirt)
+
+options:
+    state:
+        description:
+            - Set the virtual machine to either C(present), C(absent), C(running) or C(stopped).
+        default: "present"
+        choices:
+            - present
+            - absent
+            - running
+            - stopped
+        type: str
+    name:
+        description:
+            - Name of the virtual machine.
+        required: true
+        type: str
+    namespace:
+        description:
+            - Namespace where the virtual machine exists.
+        required: true
```

Comment:
I would do it optional. We could use `default` if not provided.

Reply:
Well, I think it's not good idea, the reason I did not do it even in oVirt modules is that if you for example have a VM for example `testvm` in namespace `testns`, and you want to update that vm memory lets say, if you forget to specify the namespace you may change vm in namespace `default` which is something you don't want, and with current aprroach it will fail. But we can discuss.

Reply:
Sounds reasonable